Library/Formula/global.rb

```diff
@@ -21,14 +21,16 @@ class Global < Formula
 
   option "with-exuberant-ctags", "Enable Exuberant Ctags as a plug-in parser"
   option "with-pygments", "Enable Pygments as a plug-in parser (should enable exuberent-ctags too)"
+  option "with-sqlite3", "Use SQLite3 API instead of BSD/DB API for making tag files"
```

Comment:
Maybe, yes.

Global itself uses `--with-sqlite3` as configure option and also names the option for gtags(1) `sqlite3`. So I think it makes sense to stick with that.